src/shared/utils/validation/categoryValidation.ts

```
@@ -0,0 +1,183 @@
+}
+
+/**
+ * Gets the set of valid category IDs from a kit file.
+ * Includes both standard categories and custom categories defined in the file.
+ */
+export function getValidCategoryIds(
+  customCategories: ImportedCategory[] | undefined,
+): Set<string> {
+  const validIds = new Set<string>(VALID_CATEGORIES);
+  if (customCategories) {
+    customCategories.forEach((cat) => validIds.add(cat.id));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)